### DISCUSSION TITLE
How to enable the latest user interface?

### DISCUSSION THREAD

**Discussion by nexus9xx** [PoomSmart/YTABConfig#9] [Q&A]
Which switch do I need to turn on...

**Comment by 6oo6** [comment]
enableIosRoundedSearchBar | On

DemouiSystemsClientGlobalConfigEnableModernButtonsForNative | On
uiSystemsClientGlobalConfigEnableRoundedDialogForNative	| On

uiSystemsClientGlobalConfigEnableRoundedThumbnailsForNative	| On

enableResumeToShorts | Off

enableSwipeToRemoveInPlaylistWatchEp | On

enableCinematicContainer | On
enableCinematicContainerOnClient | On

isLandscapeEngagementPanelEnabled | Off

isLandscapeEngagementPanelSwipeRightToDismissEnabled | On

replaceNextPaddleWithFastForwardButtonForSingletonVods | On
replacePreviousPaddleWithRewindButtonForSingletonVods | On

removeNextPaddleForSingletonVideos | On
removePreviousPaddleForSingletonVideos | On

**Reply by mrxx71** [reply]
i did this and almost nothing has changed, except for some rounded things such as the search bar.
Should it be looking like this?

![IMG_1665](https://user-images.githubusercontent.com/117075973/198976859-2b7735aa-abb5-49b6-90bb-cbd5bcd64d5e.PNG)
![IMG_1664](https://user-images.githubusercontent.com/117075973/198976868-796b267d-efd1-49fd-9ecf-460f4e0d18bb.PNG)
![IMG_1663](https://user-images.githubusercontent.com/117075973/198976875-c7ef938c-17ee-4eb9-93ed-d214b05de41e.PNG)

**Reply by 6oo6** [reply]
Please try darkmode! If this dont work please reset your A/B settings.

**Reply by mrxx71** [reply]
So it is supposed to have the entire new look, right? Like also the color fading behind the video and stuff? I’m resetting it now. Tried the dark mode, the darker mode and oled, but remains exactly the same

**Reply by mrxx71** [reply]
These are the copied modifications I've copied from your post. Tried it for the second time. And I'm attaching the print screens of how it looks now after the second try. I don't know if I'm missing something 


![image](https://user-images.githubusercontent.com/117075973/199060876-e5146a12-5d8a-4e08-aaa4-aaa3225e8e91.png)

![image](https://user-images.githubusercontent.com/117075973/199060930-e089a712-466b-4484-9ed6-c516d2e481ce.png)

![image](https://user-images.githubusercontent.com/117075973/199060945-a757ee2f-6cf2-4f4b-b079-35c86087032f.png)

Total: 10
YTColdConfig.enableCinematicContainer: 1
YTColdConfig.enableCinematicContainerOnClient: 1
YTColdConfig.enableIosRoundedSearchBar: 1
YTColdConfig.enableResumeToShorts: 0
YTColdConfig.isLandscapeEngagementPanelEnabled: 0
YTColdConfig.replaceNextPaddleWithFastForwardButtonForSingletonVods: 1
YTColdConfig.replacePreviousPaddleWithRewindButtonForSingletonVods: 1
YTColdConfig.uiSystemsClientGlobalConfigEnableModernButtonsForNative: 1
YTColdConfig.uiSystemsClientGlobalConfigEnableRoundedDialogForNative: 1
YTColdConfig.uiSystemsClientGlobalConfigEnableRoundedThumbnailsForNative: 1
